backend/src/main/kotlin/fr/gouv/cnsp/monitorfish/infrastructure/api/outputs/PortDataOutput.kt

```diff
@@ -3,13 +3,17 @@ package fr.gouv.cnsp.monitorfish.infrastructure.api.outputs
 import fr.gouv.cnsp.monitorfish.domain.entities.port.Port
 
 data class PortDataOutput(
-    var locode: String? = null,
-    var name: String? = null,
+    val locode: String? = null,
```

Comment:
Ces 4 props ne sont pas obligatoires pour un port par définition ?

Reply:
Nope, il n'y a pas de contrainte `not null` en base de donnée, il y a plusieurs ports où la latitude/longitude est `null` mais pour les locode/name, on pourrait rajouter p-e rajouter une contrainte dans le futur.